Plash/Menus.swift

```diff
@@ -166,6 +166,14 @@ extension AppState {
 	func updateMenu() {
 		menu.removeAllItems()
 
+		menu.addCallbackItem(
+			"Toggle \(!AppState.shared.isManuallyDisabled ? "Off" : "On")"
+		) {
+			AppState.shared.isManuallyDisabled.toggle()
```

Comment:
```suggestion
			isManuallyDisabled.toggle()
```

Same with the above.